src/main/python/main/ayab/engine/hw_test_communication_mock.py

```diff
@@ -61,7 +61,7 @@ def read_API6(self):
         if len(self.rx_msg_list) == 0:
             return None
         # else
-        res = self.rx_msg_list.popleft()  # FIFO
```

Comment:
Oops, I didn't mean for this to go into this PR, it's a partial fix for the fact that the Test dialog doesn't open in Simulation mode. No harm in keeping it here though.

Reply:
Sneaky fix